lib/result-view.coffee

```diff
@@ -227,12 +226,9 @@ transformimeJupyter.consoleTextTransform.mimetype = [
 ]
 
 transform = transformime.createTransform [
-    transformimeJupyter.PDFTransform,
     transformime.ImageTransformer,
     transformimeJupyter.SVGTransform,
     transformimeJupyter.consoleTextTransform,
-    transformimeJupyter.LaTeXTransform,
     MarkdownTransform,
     transformime.HTMLTransformer,
-    transformimeJupyter.ScriptTransform
```

Comment:
Yes. Currently if they are not included [here](https://github.com/nteract/hydrogen/blob/master/lib/kernel.coffee#L352) hydrogen will note use them.